easybuild/easyconfigs/m/MuTect/MuTect-1.1.7-Java-1.7.0_80.eb

```diff
@@ -0,0 +1,39 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# Version 1.1.7 modified by
+# Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'Tarball'
+
+name = 'MuTect'
```

Comment:
Bah. Trying to do things in a rush.

What's the least painful thing to do here?